src/main/java/com/asu/cse545/group12/controller/TransactionController.java

```diff
@@ -330,7 +330,7 @@ public ModelAndView transferAmount(@Valid @ModelAttribute("form") TransactionFor
 
 	}
 
-	@RequestMapping(value = "goBack")
+	@RequestMapping(params = "goBack", method = RequestMethod.POST)

```

Comment:
Remove this @RequestMapping(params = "goBack", method = RequestMethod.POST)

Add this: @RequestMapping(value = "goBack")